lldb/source/Plugins/Instruction/RISCV/EmulateInstructionRISCV.h

```diff
@@ -57,11 +57,12 @@ class EmulateInstructionRISCV : public EmulateInstruction {
 
   static bool SupportsThisInstructionType(InstructionType inst_type) {
     switch (inst_type) {
-    case eInstructionTypeAny:
-    case eInstructionTypePCModifying:
+    case lldb_private::eInstructionTypeAny:
```

Comment:
This method is inside a `namespace lldb_private`, please don't add `lldb_private::` specifiers.